pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -540,46 +548,56 @@ For example, given the module declaration
 
 the evaluation of @racket[(require 'm)] creates the variable @racket[x]
 and installs @racket[10] as its value. This @racket[x] is unrelated to
-any top-level definition of @racket[x].
+any top-level definition of @racket[x]. It's as
+if the variable were actually named ``x in m.''
 
-@;- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+@;------------------------------------------------------------------------
 @subsection[#:tag "module-phase"]{Phases}
 
-A module can be @tech{instantiate}d in multiple @deftech{phases}. A
-phase is an integer that, again, is effectively a prefix on the names
-of module-level definitions. A top-level @racket[require]
+The purpose of @deftech{phases} is to solve the ``eval-when'' problem that
```

Comment:
What if readers don't know the "eval-when" problem? How about shortening to:

> The purpose of phases is to separate names defined at execution time versus names defined at expansion time. A module can be .... definitions. Phase 0 is the execution-time phase.

and cutting out the reference to Flatt02 b/c it appears later on

Reply:
Sounds good, though I'd keep the citation. I couldn't understand phases without reading his paper twice.